skimage/draw/draw.py

```diff
@@ -735,15 +735,15 @@ def rectangle(origin, extent, shape=None):
            [0, 0, 0, 0, 0]], dtype=uint8)
 
     """
-    if not(len(origin) == len(extent)):
-        raise ValueError("origin and extent must have the same\
-                          number of dimensions")
-    if not(shape is None):
+    if len(origin) != len(extent):
+        raise ValueError("origin and extent must have the same"
+                         "number of dimensions")
```

Comment:
In this multi-line string, you need a space either at the end of the first line or at the start of the second line (within the quotes). I generally prefer the first option.